src/java/picard/analysis/CollectQualityYieldMetrics.java

```diff
@@ -45,12 +45,29 @@
  *
  * @author Martha Borkan
  */
+
+
 @CommandLineProgramProperties(
-        usage = "Collects quality yield metrics, a set of metrics that quantify the quality and yield of sequence data from a " +
-                "SAM/BAM input file.",
-        usageShort = "Collects a set of metrics that quantify the quality and yield of sequence data from the provided SAM/BAM",
+        usage = "CollectQualityYieldMetrics This tool collects quality yield metrics, a set of metrics that quantify " +
+                "the quality and yield of sequence data from a SAM/BAM input file.  Creates bins that group both the" +
+                "total number of bases by machine-assigned quality score and those with particular quality scores that" +
+                "have also passed filters.  Either original (OQ) the default, or the recalibrated (QUAL) base-quality" +
```

Comment:
Does this refer to Illumina machine filters, like previously? We should try to find out if these tools will work on non-Illumina data.